Aptoide/aptoide-client-v8
# [feature/ASV-1687] Errors in Search

**What does this PR do?**

   This PR refactors the whole Error view so that we reuse the same logic and the same view everywhere. It also adds error support to search.

**Database changed?**

   No

**How should this be manually tested?**

  Every error should be checked, as it was changed, but especially the new errors for search that were added. You can check this by disabling internet and searching. Then re-enable and click on retry. I'm not sure how you could test generic errors though.

**What are the relevant tickets?**

  [ASV-1687](https://aptoide.atlassian.net/browse/ASV-1687)

**Code Review Checklist**
